src/components/billingRecord/IFXCalculateBillingMonth.vue

```diff
@@ -161,7 +165,7 @@ export default {
 </script>
 
 <template>
-  <v-container v-if="!isLoading">
+  <v-container>
```

Comment:
Does this throw errors later on at line 247 or line 253?

Reply:
if there is no data in the tables, I'm not sure how a table column slot would throw an error